phy/cluster/manual/cluster_metadata.py

```diff
@@ -6,13 +6,11 @@
 # Imports
 #------------------------------------------------------------------------------
 
-from collections import defaultdict, OrderedDict, MutableMapping
+from collections import defaultdict
 from copy import deepcopy
 
-from ...utils._color import _random_color
-from ...utils._misc import _as_dict, _fun_arg_count, _as_list, _is_list
-from ...ext.six import iterkeys, itervalues, iteritems
-from ._utils import _unique, _spikes_in_clusters, UpdateInfo
+from ...utils._misc import _as_list, _is_list
```

Comment:
each time you wrote more than one . you kill a kitten and make your code less maintainable :P